src/cloudai/workloads/moe_benchmark/slurm_command_gen_strategy.py

```
@@ -0,0 +1,90 @@
+        cmd_args: MoEBenchmarkCmdArgs = tdef.cmd_args
+
+        if cmd_args.mode == "standard":
+            script_name = "benchmark.py"
+        else:
+            script_name = "benchmark_ll.py"
+
+        benchmark_script = str(PurePosixPath(cmd_args.benchmark_root) / script_name)
+
+        return ["python", benchmark_script, cmd_args.config_file_path]
+
+    def _generate_config_yaml(self, config_path: Path, cmd_args: MoEBenchmarkCmdArgs) -> None:
```

Ruff (0.15.15)

[warning] Unused method argument: `cmd_args`

(ARG002)